src/sancus-step/sancus_step.c

```diff
@@ -72,6 +75,11 @@ void __ss_start(void)
     timer_irqc(INIT_LATENCY);
 }
 
+void __ss_set_dma_attacker_delay(int delay)
```

Comment:
ideally this call shouldn't be needed I think: 

* initialization of the delay could happen automatically in `ss_init`; now if one doesn't call this function strange things will happen?
* why would one need a non-zero delay argument here?